crates/primitives-traits/src/node.rs

```diff
@@ -73,3 +73,6 @@ where
     type TxType = T::TxType;
     type Receipt = T::Receipt;
 }
+
+/// Helper adapter type for accessing [`NodePrimitives`] receipt type.
+pub type Rcpt<N> = <N as NodePrimitives>::Receipt;
```

Comment:
not a fan of abbreviations for aliases.

we will likely need a few more, so I think we should come up with a common prefix for these, perhaps just `NodeReceipt`?

Reply:
I thought we rename it when we don't use `reth_primitives::Receipt` in the code base anymore?

Reply:
ah no the trait is the problem nvm...wanted smthg short...

Reply:
`ReceiptTy`? common suffix?